src/connection-pool-party.js

```diff
@@ -484,4 +516,86 @@ export default class ConnectionPoolParty extends EventEmitter {
         err => err,
       );
   }
+
+  _prioritizePools = () => {
+    if (!this.warmedUp) {
+      debug('_prioritizePools called before warmup completed. this should not happen.');
+      return;
+    }
+    if (this._healingPromise) {
+      debug('_prioritizePools called during heal, skipping this run');
+      return;
+    }
+    if (this.pools.length === 0) {
```

Comment:
We can move the `pools.length` check into `_startPrioritizingPools`. We can prevent starting the timer if we don't have more than 1 pool.

Reply:
i'd rather keep the timer going on the off chance that somehow a warmup happened without actually having any pools, since `_startPrioritizingPools` only starts after a warmup.  i can't think of a situation where we'd have 0 pools immediately after a warmup, but it seems safer to just let the timer run and have the handler respond appropriately to the pool size

Reply:
We also don't need to have the timer for pool size of 1.
But I'm ok w/ this.

Reply:
yea, presumably if there's only one DSN, they wouldn't enable the prioritize flag